<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient_Widget</class>
 <widget class="QWidget" name="TcpClient_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="msg_lineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="msg_btn">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="username_lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>服务器IP：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="server_ip"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>端口号：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="port_lineEdit"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="enter_btn">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
